dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/UploadFrameworkDataJourney.ts

```diff
@@ -42,28 +40,32 @@ describe('As a user, I expect the dataset upload process to behave as I expect',
       let storedCompanyForManyDatasetsCompany: StoredCompany;
 
       before(function uploadOneCompanyWithoutDataAndOneCompanyWithManyDatasets() {
-        let euTaxoFinancialPreparedFixtures: Array<FixtureData<EuTaxonomyDataForFinancials>>;
-        let lksgPreparedFixtures: Array<FixtureData<LksgData>>;
-        cy.fixture('CompanyInformationWithEuTaxonomyDataForFinancialsPreparedFixtures').then(function (jsonContent) {
-          euTaxoFinancialPreparedFixtures = jsonContent as Array<FixtureData<EuTaxonomyDataForFinancials>>;
+        let euTaxoFinancialPreparedFixture: FixtureData<EuTaxonomyFinancialsData>;
+        let lksgPreparedFixture: FixtureData<LksgData>;
+        cy.fixture('CompanyInformationWithEuTaxonomyFinancialsPreparedFixtures').then(function (jsonContent) {
+          const euTaxoFinancialPreparedFixtures = jsonContent as Array<FixtureData<EuTaxonomyFinancialsData>>;
+          euTaxoFinancialPreparedFixture = getPreparedFixture(
+            'eu-taxonomy-financials-dataset-with-no-null-fields',
+            euTaxoFinancialPreparedFixtures
+          );
         });
         cy.fixture('CompanyInformationWithLksgPreparedFixtures').then(function (jsonContent) {
-          lksgPreparedFixtures = jsonContent as Array<FixtureData<LksgData>>;
+          const lksgPreparedFixtures = jsonContent as Array<FixtureData<LksgData>>;
```

Comment:
Warum ändert sich hier etwas für LkSG?

Reply:
Reines refactoring, um das Laden der fixtures in dem "before"-Block hier einheitlich und etwas leaner zu machen.

Habe ich nebenbei gemacht, während ich den Test hier repariert habe (war rot in der CI).